includes/blocks/class-sensei-block-take-course.php

```diff
@@ -29,12 +29,22 @@ public function __construct() {
 	 * @access private
 	 */
 	public function register_block() {
-		register_block_type(
-			'sensei-lms/button-take-course',
-			[
-				'render_callback' => [ $this, 'render_take_course_block' ],
-			]
-		);
+
+		/**
+		 * Filter the render callback of the Take Course block.
+		 *
+		 * In WordPress versions 5.5 and later, block type arguments can be filtered by using register_block_type_args
+		 * instead. This filter exists to support earlier versions only.
+		 *
+		 * @since 3.6.0
+		 * @hook sensei_take_course_block_type_args
+		 * @see register_block_type
+		 *
+		 * @param {array} $args The block arguments as defined by register_block_type.
+		 */
+		$block_args = apply_filters( 'sensei_take_course_block_type_args', [ 'render_callback' => [ $this, 'render_take_course_block' ] ] );
```

Comment:
Note that this filter is showing up as an action in the [hook docs](https://automattic.github.io/sensei/sensei_take_course_block_type_args.html). This is because filters need an `@return` attribute - https://github.com/Automattic/sensei/pull/3569/commits/4f98ea0b50494bf9796c67acf46a6fd0963aa125.